tests/end_to_end_tests/managed_packages_tests/conftest.py

```diff
@@ -171,37 +172,55 @@ def build(
         )
 
         if package_type == "deb":
-            # Create deb repository using 'reprepro'.
```

Comment:
We had to switch to `aptly` from `reprepro` because the latter does not support multiple version of the same package, which we need now in order to perform the upgrade.